base_classes/NXbeam.nxdl.xml

```diff
@@ -373,13 +373,8 @@
         <field name="DIRECTION" nameType="any" units="NX_TRANSFORMATION" type="NX_NUMBER">
             <doc>
                 Direction of beam vector, its value is ignored. If missing, then the beam direction is defined as [0,0,1]
-                and passes through the origin
+                and passes through the origin. Note, this field should be referenced by the parent group's ``depends_on`` field
             </doc>
-            <attribute name="transformation_type">
-                <enumeration>
-                    <item value="translation" />
```

Comment:
The base class says it can be "translation" or "rotation". So removing this would allow "DIRECTION" to be a rotation...

Reply:
I'm using the field as a reference: [`NXtransformations` lines 130-134](https://github.com/nexusformat/definitions/blob/2aede967caebc84f3335d50f1344f9b3ed2e8603/base_classes/NXtransformations.nxdl.xml#L130-L134)

Reply:
Ah ok, makes sense, it is a direction (neither translation nor rotation). Maybe mention that the attribute "transformation_type" should not be provided?

Reply:
Done